dGame/dMission/MissionTask.cpp

```diff
@@ -352,9 +343,16 @@ void MissionTask::Progress(int32_t value, LWOOBJID associate, const std::string&
 		if (info->target != gameID) {
 			break;
 		}
-
+		// Lego Universe is a great game with very easy to understand mission progression.
+		// If we are progressing a mission for shooting gallery we need to set its
+		// completion to 1 and not the targetValue beacuse that makes sense
+		// compared to all other missions and tasks :)
```

Comment:
I think the comment is funny but going forward I'd prefer a more pragmatic and serious explanation here

Reply:
I'll clean this up.  Now that I've settled from the discovery I'll remove the snarky comment.

Reply:
It's still good to leave _a_ comment in there about why the decision was made, just less subjective is already good :)